src/components/Book.js

```diff
@@ -9,25 +9,63 @@ const Book = (props) => {
   const dispatch = useDispatch();
 
   const deleteBook = (event) => {
-    const { id } = event.target.parentElement;
+    const { id } = event.target.parentElement.parentElement.parentElement;
     removeBookAPI(id, dispatch);
   };
 
   return (
-    <li id={id}>
-      <h3>
-        {title}
-        {' '}
-        {author}
-      </h3>
-      <button
-        type="button"
-        onClick={(event) => {
-          deleteBook(event);
-        }}
-      >
-        remove
-      </button>
+    <li className="book-panel" id={id}>
+      <div className="book-specs">
+        <h3 className="category">Action</h3>
+        <h2 className="title">{title}</h2>
+        <h6 className="author">{author}</h6>
+        <div className="buttonsWrapper">
+          <button className="button" type="button">
+            Comments
+          </button>
+          <div className="separator" />
+          <button
+            className="button"
+            type="button"
+            onClick={(event) => {
+              deleteBook(event);
+            }}
+          >
+            remove
+          </button>
+          <div className="separator" />
+          <button className="button" type="button">
+            Edit
+          </button>
+        </div>
+      </div>
+      <div className="panel-middle">
+        <div className="book-progress">
+          <div className="circle-wrap">
+            <div className="circle">
+              <div className="mask full">
+                <div className="fill" />
+              </div>
+              <div className="mask half">
+                <div className="fill" />
+              </div>
+              <div className="inside-circle"> 75% </div>
+            </div>
+          </div>
+          <div>
+            <h5 className="progress-percentage">75%</h5>
+            <h6 className="progress-completed">Completed</h6>
+          </div>
+        </div>
+        <div className="line" />
+        <div className="progress-chapter">
+          <div>
+            <h6 className="current-chapter">CURRENT CHAPTER</h6>
+            <h5 className="chapter">Chapter 17</h5>
+            <button className="update-button" type="button"> UPDATE PROGRESS</button>
```

Comment:
- [ ] Let's apply the correct fonts on the right side of the cards. On the template Roboto Slab is being use.

Yours:
![image](https://user-images.githubusercontent.com/79114233/141556852-56ab4939-d921-43f3-b129-23dd16ca92e8.png)

Template:
![image](https://user-images.githubusercontent.com/79114233/141557117-e62a6807-8e6c-49e5-a037-1c63f60f24dc.png)

Reply:
Ok